python-checks/src/main/java/org/sonar/python/checks/DoublePrefixOperatorCheck.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.checks;
+
+import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.PythonSubscriptionCheck;
+import org.sonar.plugins.python.api.SubscriptionContext;
+import org.sonar.plugins.python.api.tree.Expression;
+import org.sonar.plugins.python.api.tree.ParenthesizedExpression;
+import org.sonar.plugins.python.api.tree.Tree;
+import org.sonar.plugins.python.api.tree.UnaryExpression;
+
+
+@Rule(key = "S2761")
+
+public class DoublePrefixOperatorCheck extends PythonSubscriptionCheck {
+
+  private static final String MESSAGE = "Use the (\"%s\") operator just once or not at all.?";
+
+  @Override
+  public void initialize(Context context) {
+    context.registerSyntaxNodeConsumer(Tree.Kind.BITWISE_COMPLEMENT, ctx -> doubleInversionCheck(ctx, (UnaryExpression) ctx.syntaxNode()));
+    context.registerSyntaxNodeConsumer(Tree.Kind.NOT, ctx -> doubleInversionCheck(ctx, (UnaryExpression) ctx.syntaxNode()));
+  }
+
+  private static void doubleInversionCheck(SubscriptionContext ctx, UnaryExpression original) {
+    Expression invertedExpr = original.expression();
+    while (invertedExpr.is(Tree.Kind.PARENTHESIZED)) {
+      invertedExpr = ((ParenthesizedExpression) invertedExpr).expression();
+    }
+    if(invertedExpr.is(Tree.Kind.NOT, Tree.Kind.BITWISE_COMPLEMENT) && original.is(invertedExpr.getKind())) {
+      ctx.addIssue(original, String.format(MESSAGE, original.operator().value()));
+    }
```

Comment:
Regarding `BITWISE_COMPLEMENT` : I think we should check that the type of the operand is `integer` to avoid FPs when `~` operator is overloaded.
For example:

```python
class Foo:
  def __invert__(self):
     return self.bar == 42

f  = Foo()
~~f # here ~~ it might have a different semantic 
```